x-pack/plugins/cases/public/components/all_cases/translations.ts

```diff
@@ -118,7 +122,7 @@ export const FILTER_ASSIGNEES_ARIA_LABEL = i18n.translate(
   }
 );
 
-export const CLEAR_FILTERS = i18n.translate(
+export const CLEAR_ASSIGNEES_FILTERS = i18n.translate(
```

Comment:
Not related to this PR but do you mind if you change the assignees label to `Remove all assignees` as we have in the case view page?

Reply:
will do 👍

Reply:
Sorry to chime in late here. I wonder if we should leave it as `Clear filters`. My thought is because when you click the `Remove all assignees` in the case view page it does in fact remove the assignees from the case (same thing as unselecting all the assignees). Since this is a filter, I think the text `Clear filters` is more appropriate because we aren't removing any assignees. I suppose another option would be `Clear assignees filters` 🤷‍♂️ 

What do you think @mdefazio ?

Reply:
I would prefer `Clear filters`

Reply:
oh yeah, I remember we discussed it before. Sorry, my mistake.